### ISSUE TITLE
Bug: Game of the year

### ISSUE THREAD

**Issue by Lapin-Ael** [binogure-studio/city-game-studio-i18n#679]
**Describe the bug**
Happen 2 times with me on different games. When i don't have a game to select as candidate for the game of the year, in the future the game will never ask me again to be candidate.
Tryed with, and without mods, so the mods are not in cause.

**To Reproduce**
1. Don't have a game produced running while the game of the year event start for the 1st time
2. Wait and see

**Expected behavior**
The game of year event should ask me the next year to be candidate, but never will :/

**Logs**
Attach log files to this ticket:
[Uploading citygamestudio-9.log…]

**Desktop (please complete the following information):**
 - OS: [Windows 10]

**Comment by xsellier** [comment]
Hey,

Thank you for the report. I investigated and I was not able to reproduce that issue.
Do you mind sharing your savegame so I can check from your savegame what's happening?

**Comment by Lapin-Ael** [comment]
I don't know what file is it, so i gave you all files in appdata roaming.
[CityGameStudio.zip](https://github.com/binogure-studio/city-game-studio-i18n/files/7175662/CityGameStudio.zip)

**Comment by xsellier** [comment]
Just testid on my own cmputer, as you can see I started recording in Feb 2000, till March 2001.
And the game keep asking for GOTY's even if there is no game selected for the previous year.

I'm not able to reproduce the issue even with your savegames. I wonder if I understood it correctly 
https://user-images.githubusercontent.com/6616005/133598066-1bf17b16-2b27-4c41-80a1-edafc68008c4.mp4

**Comment by Lapin-Ael** [comment]
Ah... Weird ! Maybe its a bug on my side.. Anyway i count on launch a new game without mods with slow speed (the game is too fast for the phew evolving in console/features in normal speed, hard to manage). Will do that in phew days on a new computer (i'm going to vacancy) and will tell you if you want :)
Anyway, great responsivity, i will stick arround and look very closely to this game, the dev team look very nice and cool :) thumb up for you guys, i was suspicious regarding the price and lack of late game features, but will change my review soon !!

**Comment by xsellier** [comment]
I noticed, from the logs you submitted, you encountered 2 crashes (at least), I fixed one, and I'm currently working on the second one.

The crash I fixed happened when you hack a competitor. The patch is not yet available, it will be part of the next major update, which might come within few weeks.

The other crash happens the first time you launched the game, for an unknown reason, The game started loading itself 2 times. I've no idea why atm.
